app/objects/secondclass/c_executor.py

```diff
@@ -0,0 +1,87 @@
+import marshmallow as ma
+
+from app.utility.base_object import BaseObject
+
+from app.objects.secondclass.c_parser import ParserSchema
+from app.objects.secondclass.c_variation import Variation, VariationSchema
+
+
+class ExecutorSchema(ma.Schema):
+    name = ma.fields.String(missing=None)
+    platform = ma.fields.String(missing=None)
+    command = ma.fields.String(missing=None)
+    code = ma.fields.String(missing=None)
+    language = ma.fields.String(missing=None)
+    build_target = ma.fields.String(missing=None)
+    payloads = ma.fields.List(ma.fields.String(), missing=[])
```

Comment:
Using a mutable list as a default value looks concerning to me. Should that be `missing=list` or some lambda that returns a list?